www/templates/html/Media.tpl.php

```diff
@@ -128,7 +130,17 @@
                         
                     </div>  
                 </div>          
-
+                <hr>
+                
+                <?php if($getTracks){
+                    echo '<div class="mediahub-onpage-captions">';
+                    echo $savvy->render($context, 'MediaPlayer/Transcript.tpl.php');
+                    echo '</div>';
+                    }
+                ?>
+               
+                <hr>
+                <h6 class="wdn-sans-serif">Description</h6>
```

Comment:
My logic was "there wont ever be a heading underneath this!" 

Also I wanted it to be little. 

Also I originally did that over a year ago.